src/coreclr/tools/crossgen/compare.bat

```diff
@@ -1,168 +0,0 @@
-::mscorlib
```

Comment:
nit: also found a few `crossgen1` hits in few other places:
```sh
$ git grep -li crossgen1 :/

docs/design/coreclr/botr/vectors-and-intrinsics.md
src/coreclr/crossgen-corelib.proj
src/coreclr/tools/aot/ILCompiler.Diagnostics/PerfMapWriter.cs
src/coreclr/tools/aot/ILCompiler.ReadyToRun/ObjectWriter/SymbolFileBuilder.cs
src/coreclr/tools/r2rtest/Crossgen2Runner.cs
src/coreclr/tools/r2rtest/CrossgenRunner.cs
src/tasks/Crossgen2Tasks/PrepareForReadyToRunCompilation.cs
src/tasks/Crossgen2Tasks/RunReadyToRunCompiler.cs
src/tests/readytorun/crossboundarylayout/crossboundarytests.cmd
```

Reply:
I think we can remove those occurrences in a future PR. I want to keep this one focused on the native build for simplicity.

Reply:
I suspect we also have some residua in the test build infra, around files like CLRTest.Crossgen.targets, R2RTest and such. I'll float their cleanup as one of the ideas for the Quality week next week.